lib/absinthe/subscription/local.ex

```diff
@@ -89,6 +104,9 @@ defmodule Absinthe.Subscription.Local do
     |> List.wrap()
     |> Enum.map(&to_string/1)
     |> Enum.flat_map(&Absinthe.Subscription.get(pubsub, {field, &1}))
+    |> Enum.filter(fn {topic, doc} ->
+      {topic, doc}
+    end)
```

Comment:
Suggestion:
I don't think this `Enum.filter/2` gets us much in this implementation. 

Since `filter/2` returns elements that return a truthy value, it allows everything that matches this two-tuple shape to pass through. This alone isn't so bad, but if the object of this function's inclusion is to filter _out_ anything that doesn't match, we should expect to `raise` a `FunctionClauseError` whenever anything else doesn't match. 

For example:


```elixir

Erlang/OTP 25 [erts-13.1.2] [source] [64-bit] [smp:10:10] [ds:10:10:10] [async-threads:1] [jit]

Interactive Elixir (1.14.2) - press Ctrl+C to exit (type h() ENTER for help)
iex(1)> my_list = [{:two, :tuple}, {:another, :two_tuple}, {:three, :tuple, :oh_no}]
[{:two, :tuple}, {:another, :two_tuple}, {:three, :tuple, :oh_no}]
iex(2)> Enum.filter(my_list, fn {one, two} -> {one, two} end)
** (FunctionClauseError) no function clause matching in :erl_eval."-inside-an-interpreted-fun-"/1

    The following arguments were given to :erl_eval."-inside-an-interpreted-fun-"/1:

        # 1
        {:three, :tuple, :oh_no}

    (stdlib 4.1.1) :erl_eval."-inside-an-interpreted-fun-"/1
    (stdlib 4.1.1) erl_eval.erl:898: :erl_eval.eval_fun/8
    iex:2: (file)
    iex:2: (file)
```
I would suggest either adding a second clause to the anonymous function that matches everything and returns `false` OR removing this function call entirely. A filtration based on anything that matches the two-tuple signature would look something like this:

```suggestion
    |> Enum.filter(fn 
    {_topic, _doc} ->
      true
    _ -> 
      false
    end)
```

Reply:
Oh yeah I did add this when debugging, totally don't need it, will remove